client/components/login/login.test.js

```diff
@@ -264,49 +263,6 @@ describe("<Login /> interactions", () => {
           });
       });
   });
-
-  it("should execute verifyMobileNumber if mobile phone verification needed", async () => {
-    props.settings = {mobile_phone_verification: true};
-    wrapper = mountComponent(props);
-    const login = wrapper.find(Login);
-    const handleSubmit = jest.spyOn(login.instance(), "handleSubmit");
-
-    axios.mockImplementationOnce(() => {
-      return Promise.reject({
-        response: {
-          status: 401,
-          statusText: "unauthorized",
-          data: {
-            is_active: true,
-          },
-        },
-      });
-    });
-
-    expect(wrapper.exists(PhoneInput)).toBe(true);
-    expect(wrapper.find(".row.phone-number").length).toEqual(1);
-    expect(wrapper.find("#username").length).toEqual(1);
-    expect(login.state("username")).toEqual("");
-    wrapper.find("#username").simulate("change", {
-      target: {value: "+393660011333", name: "username"},
-    });
-    expect(login.state("username")).not.toEqual("");
-    wrapper
-      .find("#password")
-      .simulate("change", {target: {value: "test password", name: "password"}});
-    expect(login.state("password")).toEqual("test password");
-
-    const event = {preventDefault: () => {}};
-    wrapper.find("form").simulate("submit", event);
-    await tick();
-    expect(handleSubmit).toHaveBeenCalled();
-    const verifyMock = login.props().verifyMobileNumber.mock;
-    expect(verifyMock.calls.length).toBe(1);
-    expect(verifyMock.calls.pop()).toEqual([true]);
-    const authenticateMock = login.props().authenticate.mock;
-    expect(authenticateMock.calls.length).toBe(1);
-    expect(authenticateMock.calls.pop()).toEqual([true]);
-  });
```

Comment:
why was this test removed? Shouldn't we check that the mobile verification is initiated as expected because `setUserData` is called with `{method: "mobile_phone", is_verified: false}`?